prisma/seed.ts

```diff
@@ -23,6 +24,11 @@ async function main() {
   const daniel = await createAuthorHelper(prisma, 'Daniel');
   const nina = await createAuthorHelper(prisma, 'Nina');
 
+  //create Label
+  await createLabelHelper(prisma, 'East Africa', 'kchinnappan');
```

Comment:
nit: We've decided to keep the all lower case hyphen format for label names. Can we change `East Africa` to `region-east-africa` just to reflect how actual labels will be going forward?